guide/interactions/replying-to-slash-commands.md

```diff
@@ -55,243 +131,377 @@ Restart your bot and then send the command to a channel your bot has access to.
-Interaction responses can use masked links (e.g. `[text](http://site.com)`) and global emojis in the message content.
-:::
+The same applies to `.deferReply()` into `.editReply()`. 
 
-## Parsing options
+The following **will stay non-ephemeral**.
 
-### Command options
+```js:no-line-numbers
+await interaction.deferReply({ ephemeral: false });
+await interaction.editReply({ content: 'pong edited', ephemeral: true });
+```
 
-In this section, we'll cover how to access the values of a command's options. Let's assume you have a command that contains the following options:
+And the following **will stay ephemeral**.
 
-```js {6-13}
-const { SlashCommandBuilder } = require('@discordjs/builders');
+```js:no-line-numbers
+await interaction.deferReply({ ephemeral: true })
+await interaction.editReply({ content: 'pong edited', ephemeral: false })
+```
 
-const data = new SlashCommandBuilder()
-	.setName('ping')
-	.setDescription('Replies with Pong!')
-	.addStringOption(option => option.setName('input').setDescription('Enter a string'))
-	.addIntegerOption(option => option.setName('int').setDescription('Enter an integer'))
-	.addNumberOption(option => option.setName('num').setDescription('Enter a number'))
-	.addBooleanOption(option => option.setName('choice').setDescription('Select a boolean'))
-	.addUserOption(option => option.setName('target').setDescription('Select a user'))
-	.addChannelOption(option => option.setName('destination').setDescription('Select a channel'))
-	.addRoleOption(option => option.setName('muted').setDescription('Select a role'))
-	.addMentionableOption(option => option.setName('mentionable').setDescription('Mention something'));
+### Ephemeral followup
+
+The ephemeral state of followups are independent of the initial reply. This is because the followup is a separate message.
+
+```js:no-line-numbers
+await interaction.reply({ content: 'pong', ephemeral: false })
+await interaction.followUp({ content: 'first followup', ephemeral: true })
+await interaction.followUp({ content: 'second followup', ephemeral: true })
 ```
 
-You can `get()` these options from the `CommandInteractionOptionResolver` as shown below:
+![](./images/followup-ephemeral.png)
 
-```js
-const string = interaction.options.getString('input');
-const integer = interaction.options.getInteger('int');
-const number = interaction.options.getNumber('num');
-const boolean = interaction.options.getBoolean('choice');
-const user = interaction.options.getUser('target');
-const member = interaction.options.getMember('target');
-const channel = interaction.options.getChannel('destination');
-const role = interaction.options.getRole('muted');
-const mentionable = interaction.options.getMentionable('mentionable');
+As you saw in the previous section about [followup a deferReply](#followup-a-deferreply), the first followup to a deferral will replace the deferral instead of sending a new message. As a result, this will also affect the ephemeral state of the first followup:
```

Comment:
```suggestion
As you saw in the previous section about [follow-up a deferReply](#followup-a-deferreply), the first follow-up to a deferral will replace the deferral instead of sending a new message. As a result, this will also affect the ephemeral state of the first follow-up:
```